src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/contactperson/ContactPersonResourceIntTest.kt

```diff
@@ -712,6 +712,11 @@ class ContactPersonResourceIntTest : IntegrationTestBase() {
             whenCreated = LocalDateTime.parse("2024-01-01T10:00").minusMinutes(it.toLong()),
```

Comment:
Not sure if these tests should actually be running against `2024-01-01T00:10` instead and then they would span the beginning of the day.  As it is they are at 9.40 to 10.00.

Reply:
maybe better would be to do `2024-01-02T10:00` and then `minusHours`, which would span the test period...